common/Resources/ti.main.js

```diff
@@ -23,7 +23,9 @@ try {
 }
 
 // Load JS language polyfills
-import '@babel/polyfill';
+import 'core-js/stable';
```

Comment:
To avoid the web exclusion stuff, maybe change this to `import 'core-js/es';`? That includes only language polyfills, and not web ones.See https://github.com/zloirock/core-js#commonjs-api

Reply:
That reduced the size by around 100kb. There looks to be a 60kb increase over the current size but I feel that should be acceptable